From 88915688b20416470e7a19bada4860e6645314d5 Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Mon, 17 Apr 2023 12:20:05 +0100
Subject: [PATCH 1/2] gtk: Add more URLs to AppStream metainfo

This covers all the URL types from the AppStream specification.

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url
---
 gtk/transmission-gtk.metainfo.xml.in | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/gtk/transmission-gtk.metainfo.xml.in b/gtk/transmission-gtk.metainfo.xml.in
index fb7d59dfc..15a6656b9 100644
--- a/gtk/transmission-gtk.metainfo.xml.in
+++ b/gtk/transmission-gtk.metainfo.xml.in
@@ -29,6 +29,14 @@ Copyright 2017 Endless Mobile, Inc.
     </p>
   </description>
   <url type="homepage">https://transmissionbt.com/</url>
+  <url type="bugtracker">https://github.com/transmission/transmission/issues</url>
+  <url type="faq">https://transmissionbt.com/help/gtk/4.0x/html/FAQ.html</url>
+  <url type="help">https://transmissionbt.com/help/gtk/4.0x/index.html</url>
+  <url type="donation">https://transmissionbt.com/donate</url>
+  <url type="translate">https://explore.transifex.com/transmissionbt/transmissionbt/</url>
+  <url type="contact">https://forum.transmissionbt.com/</url>
+  <url type="vcs-browser">https://github.com/transmission/transmission</url>
+  <url type="contribute">https://github.com/transmission/transmission/blob/main/CONTRIBUTING.md</url>
   <screenshots>
     <screenshot type="default">
       <image type="source">https://raw.githubusercontent.com/transmission/transmission/main/gtk/screenshots/a.png</image>
-- 
2.30.2

